conda-recipes/llvmdev/build.sh

```diff
@@ -93,10 +100,3 @@ if [[ $ARCH == 'x86_64' ]]; then
    bin/opt -S -vector-library=SVML -mcpu=haswell -O3 $RECIPE_DIR/numba-3016.ll | bin/FileCheck $RECIPE_DIR/numba-3016.ll || exit $?
 fi
 
-# run the tests, skip some on linux-32
-cd ../test
-if [[ $ARCH == 'i686' ]]; then
-    ../build/bin/llvm-lit -vv Transforms Analysis CodeGen/X86
-else
-    ../build/bin/llvm-lit -vv Transforms ExecutionEngine Analysis CodeGen/X86
-fi
```

Comment:
What is the reason for removing these tests? If it's because they are in `build.sh` perhaps they should be moved to `run_test.sh`?

Reply:
We are having trouble getting any test to work. Deferring adding the test back to another PR given that this PR has passed buildfarm multiple times.

Reply:
Please could a new issue be opened to track this so it's visible as to what is failing/has been tried already. I also think it is ok on this occasion to defer this work to another patch given how many runs through the build farm this has had. Many thanks.

CC @apmasell.